ManualTimeLogger.Domain.Tests/ManualTimeLogger.Domain.Tests.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\NUnit3TestAdapter.3.16.1\build\net35\NUnit3TestAdapter.props" Condition="Exists('..\packages\NUnit3TestAdapter.3.16.1\build\net35\NUnit3TestAdapter.props')" />
```

Comment:
Waarom is de NUnit3TestAdapter toegevoegd?

Reply:
Dit heeft VS nodig om tests te kunnen runnen in het Test Window - voor zover ik heb kunnen achterhalen iig. Hoe dit precies zit, weet ik nog niet. Welke IDE gebruik jij hiervoor?

Reply:
Akkoord. Ook VS. Ik draai ze met Resharper.